pynamodb/connection/base.py

```diff
@@ -616,6 +617,23 @@ def create_table(self,
             raise TableError("Failed to create table: {0}".format(e), e)
         return data
 
+    def update_time_to_live(self, table_name, ttl_attribute_name):
+        """
+        Performs the DeleteTable operation
+        """
+        operation_kwargs = {
+            TABLE_NAME: table_name,
+            TIME_TO_LIVE_SPECIFICATION: {
+                ATTR_NAME: ttl_attribute_name,
+                ENABLED: True,
+            }
+        }
+        try:
+            data = self.dispatch(UPDATE_TIME_TO_LIVE, operation_kwargs)
```

Comment:
nit -
```suggestion
            return self.dispatch(UPDATE_TIME_TO_LIVE, operation_kwargs)
```

Reply:
thought it was bad practice to return in a try catch?

Will change..

Reply:
Never thought it a bad practice -- but I'm willing to be convinced otherwise. If anything, I usually clean up variables that add indirection without making code clearer:
e.g.
```
result = foo()
return result
```
to
```
return foo()
```